backend/app/main.py

```
@@ -0,0 +1,108 @@
+
+@app.get("/")
+def read_root():
+    return {"message": "AI Book Reader API", "version": "1.0.0"}
+
+@app.get("/health")
+def health_check():
+    return {"status": "healthy"}
+
+# Notes endpoints
+@app.post("/notes/", response_model=schemas.Note)
+def create_note(note: schemas.NoteCreate, db: Session = Depends(get_db)):
```

Ruff (0.13.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)